internal/server/v1/firehose/mappings.go

```diff
@@ -39,10 +39,12 @@ const (
 	labelDescription = "description"
 )
 
+const logSinkTTL = 24 * time.Hour
```

Comment:
let's create a new file called `constants.go` where we put `logSinkTTL` and other constant that are used by multiple files.